tests/build/build_templates.go

```diff
@@ -401,6 +408,20 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 				}
 			})
 
+			It("image manifest mediaType is correct", Label(buildTemplatesTestLabel), func() {
+				builtImage := build.GetBinaryImage(pr)
+				if scenario.CheckDockerManifestListMediaType {
+					// Check for docker mediaType
+					Expect(build.GetBuiltImageManifestMediaType(builtImage)).Should(Equal(build.MediaTypeDockerManifestList), "mediaType of the image manifest is not of type docker.manifest.list")
```

Comment:
Do you want to have another scenario to check for `application/vnd.docker.distribution.manifest.v2+json` which is similar to the OCI Image Manifest?
https://github.com/opencontainers/image-spec/blob/main/media-types.md#applicationvndociimagemanifestv1json